client/state/ui/action-log/reducer.js

```diff
@@ -12,22 +11,33 @@ import {
 	GUIDED_TOUR_UPDATE,
 	THEMES_RECEIVE,
 	PREVIEW_IS_SHOWING,
-	ROUTE_SET,
 } from 'state/action-types';
 
-const isRelevantActionType = includes.bind( null, [
+const isDesignRoute = ( { path } ) => 0 === path.indexOf( '/design' );
```

Comment:
That's true, but I'm not sure how to solve it _right now_. I'm leaning towards leaving it here for now and solving it in another PR.